### PR TITLE
PW-3903 Added configuration "isCardHolderNameRequired" for Credit card holder name

### PR DESCRIPTION
**Pull request by saquibsayyad**
**Description**
Added configuration "isCardHolderNameRequired" to make card holder name field mandatory or not.

**Tested scenarios**
Credit card payment with and without card holder name.

### CODE REVIEW

File: adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/_ui/responsive/common/js/adyen.checkout.js [hunk 1]
```diff
@@ -76,6 +76,8 @@ var AdyenCheckoutHybris = (function () {
 
                 if ( isInvalidCard ) {
                     window.alert( 'This ' + this.getCardType() + ' card is not allowed' );
```
Comment (saquibsayyad) [inline adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/_ui/responsive/common/js/adyen.checkout.js:78]: @rkewlani @martinsrenato Do you think we should change this message to fill all the details instead?

Comment (martinsrenato) [reply adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/_ui/responsive/common/js/adyen.checkout.js:78]: I think so.
It can be invalid because of missing fields, like you said, but also because the card number typed is invalid or from an unsupported brand. So we could have specific messages for both cases, if there's any way to identify that through the component validation, or a generic message that works for both.

File: adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/_ui/responsive/common/js/adyen.checkout.js [hunk 2]
```diff
@@ -125,6 +127,7 @@ var AdyenCheckoutHybris = (function () {
                 if (!this.afterPay.state.isValid) {
                     window.alert("Please fill all the details");
                     this.afterPay.showValidation();
+                    document.getElementById("afterpay-container").scrollIntoView();
```
Comment (martinsrenato) [inline adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/_ui/responsive/common/js/adyen.checkout.js:130]: This seems unrelated to the rest, is it needed?

File: adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/_ui/responsive/common/js/adyen.checkout.js (hunk 1 above)
Comment (saquibsayyad) [reply adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/_ui/responsive/common/js/adyen.checkout.js:78]: For Unsupported cards it shows message in the component itself. 

Screen Shot 2021-02-18 at 3 33 27 PM

So i think we can show a generic message like "Please check your card details" for the whole form.

File: adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/_ui/responsive/common/js/adyen.checkout.js (hunk 2 above)
Comment (saquibsayyad) [reply adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/_ui/responsive/common/js/adyen.checkout.js:130]: I added this in the places where we are calling `showValidation()` to scroll into the area where error message is shown, makes it easier for user.

File: adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/_ui/responsive/common/js/adyen.checkout.js (hunk 1 above)
Comment (martinsrenato) [reply adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/_ui/responsive/common/js/adyen.checkout.js:78]: I agree

**Review by Aleffio — APPROVED** [review]

File: adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/_ui/responsive/common/js/adyen.checkout.js (hunk 1 above)
Comment (saquibsayyad) [reply adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/_ui/responsive/common/js/adyen.checkout.js]: Message is updated now.

**Review by martinsrenato — APPROVED** [review]

**Review by rkewlani — APPROVED** [review]
